schedulers_definition.py

```diff
@@ -54,32 +54,32 @@ def getBigtestBuilderNames(props):
     return []
 
 
-@util.renderer
-def getInstallBuilderNames(props):
-    builderName = str(props.getProperty("parentbuildername"))
-
-    for b in builders_install:
+def getBuilderNames(builderName, builders_list):
+    builders = []
+    for b in builders_list:
         if builderName in b:
-            builders = [b]
+            builders.append(b)
             if "rhel" in builderName:
                 builders.append(b.replace("rhel", "almalinux"))
                 builders.append(b.replace("rhel", "rockylinux"))
-            return builders
-    return []
+            if "sles-1505" in builderName or "opensuse-1505" in builderName:
+                builders.append(b.replace("1505", "1506"))
+            break
+    return builders
```

Comment:
Nice touch here, rather than returning an empty list if the parent builder doesn't have any install / upgrade builders defined.